NitroxClient/Debuggers/Drawer/UnityUI/RectTransformDrawer.cs

```diff
@@ -0,0 +1,208 @@
+
+    public Type[] ApplicableTypes { get; } = { typeof(RectTransform) };
+
+    public void Draw(object target)
+    {
+        switch (target)
+        {
+            case RectTransform rectTransform:
+                DrawRectTransform(rectTransform);
+                break;
+        }
+    }
+
+    private static void DrawRectTransform(RectTransform rectTransform)
+    {
+        using (new GUILayout.VerticalScope())
+        {
+            //TODO: Implement position display like the Unity editor
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Local Position", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rectTransform.localPosition = VectorDrawer.DrawVector3(rectTransform.localPosition, VECTOR_MAX_WIDTH);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Local  Rotation", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rectTransform.localRotation = Quaternion.Euler(VectorDrawer.DrawVector3(rectTransform.localRotation.eulerAngles, VECTOR_MAX_WIDTH));
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Local  Scale", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rectTransform.localScale = VectorDrawer.DrawVector3(rectTransform.localScale, VECTOR_MAX_WIDTH);
+            }
+
+            GUILayout.Space(20);
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Size", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                rectTransform.sizeDelta = VectorDrawer.DrawVector2(rectTransform.sizeDelta, VECTOR_MAX_WIDTH);
+            }
+
+            using (new GUILayout.HorizontalScope())
+            {
+                GUILayout.Label("Anchor", NitroxGUILayout.DrawerLabel, GUILayout.Width(LABEL_WIDTH));
+                NitroxGUILayout.Separator();
+                AnchorMode anchorMode = VectorToAnchorMode(rectTransform.anchorMin, rectTransform.anchorMax);
+
+                if (anchorMode == AnchorMode.NONE)
+                {
+                    VectorDrawer.DrawVector2(rectTransform.anchorMin, VECTOR_MAX_WIDTH * 0.5f);
+                    VectorDrawer.DrawVector2(rectTransform.anchorMax, VECTOR_MAX_WIDTH * 0.5f);
+                }
+                else
+                {
+                    anchorMode = NitroxGUILayout.EnumPopup(anchorMode, VECTOR_MAX_WIDTH);
+
+                    // Vector2[] anchorVectors = AnchorModeToVector(anchorMode);
+                    // rectTransform.anchorMin = anchorVectors[0];
+                    // rectTransform.anchorMax = anchorVectors[1];
```

Comment:
If this isnt needed can we remove?

Reply:
There is currently a parsing bug which corrupts the Anchor value. Therefore this is technically needed for full support but not ready for now.